src/site/twitch.tv/modules/chat/ChatModule.vue

```diff
@@ -353,6 +353,18 @@ export const config = [
 		disabledIf: () => !useConfig("highlights.basic.mention").value,
 		defaultValue: true,
 	}),
+	declareConfig<boolean>("highlights.basic.suspicious_user", "TOGGLE", {
+		path: ["Highlights", "Built-In"],
+		label: "Show Suspicious User Highlights",
+		hint: "Whether or not to highlight users who are marked as a suspicious user",
+		defaultValue: true,
+	}),
+	declareConfig<boolean>("highlights.basic.low_trust_user", "TOGGLE", {
+		path: ["Highlights", "Built-In"],
+		label: "Show Low Trust User Highlights",
+		hint: "Whether or not to highlight users who are marked as a low trust user",
+		defaultValue: true,
+	}),
```

Comment:
"Low Trust User" and "Suspicious User" are synonymous. The former is just the technical term used by Twitch. The only differentiation is "Monitored" and "Restricted".

Reply:
Should both of these options be moved to "Chat > Moderation" instead and change the Restricted to opacity so it's always visible?

Reply:
Keep it under highlights for now. Ideally these things should actually sync up with the native highlights menu from twitch, but we don't have hooks into this yet. I may look into it sometime if no one else does